### PR TITLE
Added Milk and Cookies Presence

### PR DESCRIPTION
**Pull request by JeffreyDeveloper**
Presence for website Milk and Cookies

### CODE REVIEW

File: websites/M/Milk and Cookies/dist/metadata.json [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+    "id": "540294552372772884"
+  },
+  "url": "milkandcookies.games",
+  "description": {
+    "es": "¡Comunidad de Discord para todos!"
+  },
+  "service": "Milk and Cookies",
+  "version": "1.0",
+  "logo": "https://cdn.discordapp.com/icons/740677919919046788/52922d16a00f0a8de2b9d5cbcec3f8fd.webp?size=2048",
+  "thumbnail": "https://cdn.discordapp.com/icons/740677919919046788/52922d16a00f0a8de2b9d5cbcec3f8fd.webp?size=2048",
```
Comment (Bas950) [inline websites/M/Milk and Cookies/dist/metadata.json:14]: upload to imgur

File: websites/M/Milk and Cookies/dist/metadata.json [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+  "author": {
+    "name": "Jeffrey.",
+    "id": "540294552372772884"
+  },
+  "url": "milkandcookies.games",
+  "description": {
+    "es": "¡Comunidad de Discord para todos!"
+  },
+  "service": "Milk and Cookies",
+  "version": "1.0",
```
Comment (Bas950) [inline websites/M/Milk and Cookies/dist/metadata.json:12]: version 1.0.0

File: websites/M/Milk and Cookies/dist/metadata.json [hunk 3]
```diff
@@ -0,0 +1,25 @@
+{
+  "$schema": "https://schemas.premid.app/metadata/1.0",
+  "author": {
+    "name": "Jeffrey.",
+    "id": "540294552372772884"
+  },
+  "url": "milkandcookies.games",
+  "description": {
+    "es": "¡Comunidad de Discord para todos!"
```
Comment (Bas950) [inline websites/M/Milk and Cookies/dist/metadata.json:9]: you also need an english description

File: websites/M/Milk and Cookies/dist/metadata.json [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+  "logo": "https://cdn.discordapp.com/icons/740677919919046788/52922d16a00f0a8de2b9d5cbcec3f8fd.webp?size=2048",
+  "thumbnail": "https://cdn.discordapp.com/icons/740677919919046788/52922d16a00f0a8de2b9d5cbcec3f8fd.webp?size=2048",
+  "color": "#FF0098",
+  "tags": [
+    "discord",
+    "server",
+    "aesthetic",
+    "milk and cookies",
+    "cookies and milk",
+    "galletas y leche"
```
Comment (Bas950) [inline websites/M/Milk and Cookies/dist/metadata.json:22]: cant have spaces in tags

File: websites/M/Milk and Cookies/presence.ts [hunk 5]
```diff
@@ -0,0 +1,33 @@
+var presence = new Presence({
+    clientId: "758753662079729764"
+})
+
+var browsingStamp = Math.floor(Date.now() / 1000);
+
```
Comment (Bas950) [inline websites/M/Milk and Cookies/presence.ts:6]: no var (see codacy errors)

**Review by Bas950 — CHANGES_REQUESTED** [review]
Dont upload JS file

File: websites/M/Milk and Cookies/dist/metadata.json [hunk 6]
```diff
@@ -0,0 +1,26 @@
+{
+  "$schema": "https://schemas.premid.app/metadata/1.0",
+  "author": {
+    "name": "Jeffrey.",
+    "id": "540294552372772884"
+  },
+  "url": "milkandcookies.games",
+  "description": {
+    "es": "¡Comunidad de Discord para todos!".
+    "en": "Discord Community for everyone!"
```
Comment (Bas950) [inline websites/M/Milk and Cookies/dist/metadata.json:10]: , not .

**Review by Bas950 — CHANGES_REQUESTED** [review]

File: websites/M/Milk and Cookies/dist/metadata.json [hunk 7]
```diff
@@ -0,0 +1,26 @@
...
+  },
+  "url": "milkandcookies.games",
+  "description": {
+    "es": "¡Comunidad de Discord para todos!",
+    "en": "Discord Community for everyone!"
+  },
+  "service": "Milk and Cookies",
+  "version": "1.0.0",
+  "logo": "https://imgur.com/g2C5xmp",
+  "thumbnail": "https://imgur.com/g2C5xmp",
```
Comment (ririxichan) [inline websites/M/Milk and Cookies/dist/metadata.json:15]: Change to https://i.imgur.com/g2C5xmp.png

**Review by ririxichan — CHANGES_REQUESTED** [review]
change to i.imgur.com

File: websites/M/Milk and Cookies/dist/metadata.json [hunk 8]
```diff
@@ -11,8 +11,8 @@
   },
   "service": "Milk and Cookies",
   "version": "1.0.0",
-  "logo": "https://imgur.com/g2C5xmp",
-  "thumbnail": "https://imgur.com/g2C5xmp",
+  "logo": "https://i.imgur.com/g2C5xmp",
+  "thumbnail": "https://i.imgur.com/g2C5xmp",
```
Comment (ririxichan) [inline websites/M/Milk and Cookies/dist/metadata.json:15]: add .png to end in both

**Review by ririxichan — CHANGES_REQUESTED** [review]

**Review by ririxichan — APPROVED** [review]

File: websites/M/Milk and Cookies/presence.ts [hunk 9]
```diff
@@ -0,0 +1,33 @@
+const presence = new Presence({ 
+  clientId: "758753662079729764" 
+});
+
+const browsingStamp = Math.floor(Date.now() / 1000);
```
Comment (ririxichan) [inline websites/M/Milk and Cookies/presence.ts:5]: merge consts (just noticed it)

**Review by ririxichan — CHANGES_REQUESTED** [review]

File: websites/M/Milk and Cookies/presence.ts [hunk 10]
```diff
@@ -0,0 +1,32 @@
+const presence = new Presence({ 
+  clientId: "758753662079729764" 
+});
+
+presence.on("UpdateData", async () => {
+    const presenceData: PresenceData = {
+		largeImageKey: "icon"
+	};
+  browsingStamp = Math.floor(Date.now() / 1000);
```
Comment (ririxichan) [inline websites/M/Milk and Cookies/presence.ts:9]:
```javascript
const presence = new Presence({ 
  clientId: "758753662079729764" 
}),
  browsingStamp = Math.floor(Date.now() / 1000);
```
or
```javascript
const presenceData: PresenceData = {
		largeImageKey: "icon"
	},
  browsingStamp = Math.floor(Date.now() / 1000);
```

**Review by ririxichan — CHANGES_REQUESTED** [review]

File: websites/M/Milk and Cookies/dist/metadata.json [hunk 11]
```diff
@@ -0,0 +1,26 @@
...
+  "url": "milkandcookies.games",
+  "description": {
+    "es": "¡Comunidad de Discord para todos!",
+    "en": "Discord Community for everyone!"
+  },
+  "service": "Milk and Cookies",
+  "version": "1.0.0",
+  "logo": "https://i.imgur.com/g2C5xmp.png",
+  "thumbnail": "https://i.imgur.com/g2C5xmp.png",
+  "color": "#FF0098",
```
Comment (Bas950) [inline websites/M/Milk and Cookies/dist/metadata.json:16]: thumbnail should be a promotional image or a screenshot of the website homepage

**Review by Bas950 — CHANGES_REQUESTED** [review]
Upload proof of the presence working

File: websites/M/Milk and Cookies/presence.ts [hunk 12]
```diff
@@ -0,0 +1,33 @@
...
+  clientId: "758753662079729764" 
+}),
+      browsingStamp = Math.floor(Date.now() / 1000);
+
+presence.on("UpdateData", async () => {
+    const presenceData: PresenceData = {
+		largeImageKey: "icon"
+	};
+          
+
```
Comment (ririxichan) [inline websites/M/Milk and Cookies/presence.ts:11]: delete blank line

**Review by ririxichan — CHANGES_REQUESTED** [review]

**Comment by JeffreyDeveloper** [comment]
image

**Review by Bas950 — APPROVED** [review]

**Review by ririxichan — APPROVED** [review]

**Review by Alanexei — APPROVED** [review]

**Pull request merged**